scripts/fetch-iana.js

```diff
@@ -213,13 +215,13 @@ function extractTemplateExtensions (body) {
 }
 
 async function get (type, options) {
-  const res = await got('https://www.iana.org/assignments/media-types/' + encodeURIComponent(type) + '.csv')
+  const res = await request('https://www.iana.org/assignments/media-types/' + encodeURIComponent(type) + '.csv')
 
   if (res.statusCode !== 200) {
     throw new Error('got status code ' + res.statusCode + ' from ' + type)
   }
 
-  const mimes = await toArray(parser(res.body))
+  const mimes = await toArray(parser(await res.body.text()))
```

Comment:
```suggestion
  const responseText = await res.body.text();
  const mimes = await toArray(parser(responseText));
```

I believe this approach improves readability and makes the code easier to maintain.

Reply:
Done